var/spack/repos/builtin/packages/amdscalapack/package.py

```diff
@@ -32,7 +32,7 @@ class Amdscalapack(ScalapackBase):
         'build_type',
         default='Release',
         description='CMake build type',
-        values=('Release', 'RelWithDebInfo'))
+        values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
```

Comment:
I think you can completely delete this variant, since it's now the same as in the base class

Reply:
Ah, I didn't notice it was inheriting from another package. Good point.